framework/db/Connection.php

```diff
@@ -618,9 +625,10 @@ protected function initConnection()
      */
     public function createCommand($sql = null, $params = [])
     {
-        $command = new Command([
+        $command = \Yii::createObject([
```

Comment:
We have to be sure, that `$command` is instance of `\yii\db\Command`.
Maybe, `if (!$command instanceof \yii\web\Command) { throw new InvalidConfigException('...'); }` will be enough